doc/sphinxman/source/oeprop.rst

```diff
@@ -113,12 +109,12 @@ Properties evaluated on a grid
 
 Certain properties may be evaluated a user-specified grid points.  The grid
 points are completely arbitrary and are specified by providing a file called
-grid.dat containing the x,y,z values for each point in order::
+grid.dat containing the x,y,z values seperated with spacesfor each point in order::
```

Comment:
Updated.

Cheers,
Daniel Smith



> On Apr 22, 2016, at 11:30 AM, Lori A. Burns <notifications@github.com> wrote:
> 
> In doc/sphinxman/source/oeprop.rst <https://github.com/psi4/psi4/pull/365#discussion_r60757600>:
> 
> > @@ -113,12 +109,12 @@ Properties evaluated on a grid
> >  
> >  Certain properties may be evaluated a user-specified grid points.  The grid
> >  points are completely arbitrary and are specified by providing a file called
> > -grid.dat containing the x,y,z values for each point in order::
> > +grid.dat containing the x,y,z values seperated with spacesfor each point in order::
> "separated with spaces for"
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/psi4/psi4/pull/365/files/227dda60bb791070bdccca2deb87f90f6f57aab3#r60757600>